virt/release_notes/virt-4-15-release-notes.adoc

```diff
@@ -85,14 +85,14 @@ This release adds new features and enhancements related to the following compone
 [id="virt-4-15-networking"]
 === Networking
 
-//CNV-30170: NetworkPolicies filtering using IP blocks with OVN-Kubernetes
-
-//CNV-30164: NIC hot plug feature is now GA. This was TP for 4.14. Moved the release note from the TP section to here.
-* You can xref:../../virt/vm_networking/virt-hot-plugging-network-interfaces.adoc#virt-hot-plugging-network-interfaces[hot plug a bridge network interface] to a running virtual machine (VM). Hot plugging and hot unplugging is supported only for VMs created with {VirtProductName} 4.14 or later.
-
-//CNV-30167: SR-IOV NIC hot plug
+//CNV-30164 and CNV-30167: NIC hot plug feature is now GA. This was TP for 4.14. Moved the release note from the TP section to here. Covers both bridge and SR-IOV NICs
+* You can xref:../../virt/vm_networking/virt-hot-plugging-network-interfaces.adoc#virt-hot-plugging-network-interfaces[hot plug a secondary network interface] to a running VM. Hot plugging and hot unplugging is supported only for VMs created with {VirtProductName} 4.14 or later. Hot unplugging is not supported for Single Root I/O Virtualization (SR-IOV) interfaces.
```

Comment:
Suggestion to spell out Virtual Machine (especially on the first mention in this section so far):
```suggestion
* You can xref:../../virt/vm_networking/virt-hot-plugging-network-interfaces.adoc#virt-hot-plugging-network-interfaces[hot plug a secondary network interface] to a running virtual machine (VM). Hot plugging and hot unplugging is supported only for VMs created with {VirtProductName} 4.14 or later. Hot unplugging is not supported for Single Root I/O Virtualization (SR-IOV) interfaces.
```